From 8aa96744f1ea7b6df860b7d25a0f2347d257115b Mon Sep 17 00:00:00 2001
From: Emil Medve <Emilian.Medve@freescale.com>
Date: Mon, 26 Oct 2009 14:24:32 -0500
Subject: [PATCH] P4080/MAC: Use the "fixed-link" PHY support

This eliminates the platform specific code from the driver

Signed-off-by: Emil Medve <Emilian.Medve@Freescale.com>
[Kevin: Original patch taken from Freescale p4080 SDK 2.0 ISO image.
Apply cleanly to kernel 2.6.27]
Integrated-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/dpa/mac-api.c |   13 -------------
 1 files changed, 0 insertions(+), 13 deletions(-)

diff --git a/drivers/net/dpa/mac-api.c b/drivers/net/dpa/mac-api.c
index a773ec0..cd96378 100644
--- a/drivers/net/dpa/mac-api.c
+++ b/drivers/net/dpa/mac-api.c
@@ -258,7 +258,6 @@ static int __cold change_promisc(struct mac_device *mac_dev)
 	return _errno;
 }
 
-#ifndef CONFIG_P4080_SIM
 static void adjust_link(struct net_device *net_dev)
 {
 	struct dpa_priv_s *priv = netdev_priv(net_dev);
@@ -349,18 +348,6 @@ static int xgmac_init_phy(struct net_device *net_dev)
 	return 0;
 }
 
-#else
-static int dtsec_init_phy(struct net_device *net_dev)
-{
-	return 0;
-}
-
-static int xgmac_init_phy(struct net_device *net_dev)
-{
-	return 0;
-}
-#endif /* !CONFIG_P4080_SIM */
-
 static int __cold uninit(struct mac_device *mac_dev)
 {
 	int			 _errno, __errno;
-- 
1.6.0.4

